From 450befffa1e96ff8c1672c17532e94eb804dda2d Mon Sep 17 00:00:00 2001
Message-ID: <450befffa1e96ff8c1672c17532e94eb804dda2d.1732384228.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732384228.git.sharlatanus@gmail.com>
References: <cover.1732384228.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 21 Nov 2024 17:10:07 +0000
Subject: [PATCH 35/46] gnu: python-roman-datamodels: Update to 0.22.0.

* gnu/packages/astronomy.scm (python-roman-datamodels): Update to 0.22.0.
[native-inputs]: Add python-setuptools and python-wheel.

Change-Id: I7cfc5aed379c9056f93e1214ec5b06f56edced74
---
 gnu/packages/astronomy.scm | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index d793a63889..ac3b00f685 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -6359,13 +6359,13 @@ (define-public python-radiospectra
 (define-public python-roman-datamodels
   (package
     (name "python-roman-datamodels")
-    (version "0.21.0")
+    (version "0.22.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "roman_datamodels" version))
        (sha256
-        (base32 "00hwyxixdk3i48nhqnn7pnkkp3fran6fn8kzwx2w4bjkwv8cd126"))))
+        (base32 "0n50pgs7y0nmrx86qlny27mas2zgx708gm3xfkl7an2nwkdfwfld"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -6376,6 +6376,14 @@ (define-public python-roman-datamodels
           (add-after 'unpack 'set-env
             (lambda _
               (setenv "HOME" "/tmp"))))))
+    (native-inputs
+     (list python-pytest
+           python-pytest-doctestplus
+           python-pytest-env
+           python-pytest-xdist
+           python-setuptools
+           python-setuptools-scm
+           python-wheel))
     (propagated-inputs
      (list python-asdf
            python-asdf-astropy
@@ -6384,12 +6392,6 @@ (define-public python-roman-datamodels
            python-numpy
            python-psutil
            python-rad))
-    (native-inputs
-     (list python-pytest
-           python-pytest-doctestplus
-           python-pytest-env
-           python-pytest-xdist
-           python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/roman_datamodels")
     (synopsis "Roman Datamodels Support")
     (description
-- 
2.46.0

